core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.controller.ts

```diff
@@ -43,6 +43,35 @@ angular.module('oppia').controller(
       $scope.activeSuggestion = suggestionIdToSuggestion[
         $scope.activeSuggestionId];
       $scope.subheading = subheading;
+      $scope.isEditing = false;
+      $scope.isTranslationUpdated = false;
+      $scope.isAccpeted = false;
+      $scope.HTML_SCHEMA = {
+        type: 'html',
+        ui_config: {
+          hide_complex_extensions: 'true'
+        }
+      };
+      $scope.editedContent = {
```

Comment:
Why does this have to be an object? Can't we set this directly to the html string?

Reply:
Actually it gave an error such that `TypeError: Cannot set property 'html' of undefined`. Hence I used it as an object.

Reply:
Did you update all the references of `editedContent.html` to `editedContent` when you did this? If not, then that error would be expected. Also, if you change it, the name should probably be updated to something like `editedTranslationHtml`.

Reply:
I initialise the object in this way and when the translation is changed, the changed value is assigned to `editedContent.html`

Reply:
Right, you mean line 32 in core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review.directive.html?

`<schema-based-editor schema="HTML_SCHEMA" local-value="editedContent.html" ng-if="startedEditing">`

You updated the reference in the above line as well when you tried this?

Reply:
@chris7716 -- did you see this comment? It's still unreplied to.